### ISSUE TITLE
Document differences between linked ressources

### ISSUE THREAD

**Issue by TobiasNx** [hbz/lobid-resources#1400]
We have a couple of elements that link other ressources mainly within lobid or zbd, but there is no real documentation of this,
some are documented here: https://lobid.org/resources/api

`related`: no documentation
```
   {
      "container" : "@set",
      "label" : "verwandte Ressource",
      "multilangLabel" : {},
      "name" : "related",
      "uri" : "http://purl.org/dc/terms/relation"
   }
```

`isPartOf`: Enthält Informationen zum Erscheinen der Ressource als Teil eines übergeordneten Werks (eines, mehrbändigen Werks, einer Serie oder eines Periodikums). 

```
   {
      "name" : "isPartOf",
      "container" : "@set",
      "multilangLabel" : {},
      "uri" : "http://purl.org/lobid/lv#isPartOf",
      "label" : "Bestandteil von"
   }
```

`supplement`:  Enthält URIs von Beilagen zur Ressource.
```
   {
      "container" : "@set",
      "name" : "supplement",
      "multilangLabel" : {},
      "uri" : "http://id.loc.gov/ontologies/bibframe/supplement",
      "label" : "Beilage"
   }
```

`sameAs`: Enthält URIs von anderen Ressourcne, die mit der beschriebenen identisch sind. Verlinkte Quellen sind derzeit Culturegraph, ZDB-Linked-Data-Service und WorldCat.
```

   {
      "container" : "@set",
      "name" : "sameAs",
      "multilangLabel" : {},
      "label" : "Identisch zu",
      "uri" : "http://www.w3.org/2002/07/owl#sameAs"
   }
```

`containedIn`: Dies ist ein Aufsatz-spezifisches Feld. Es enthält die URI der übergeordneten Ressource (meist eine Aufsatzsammlung oder Zeitschrift), in der ein Aufsatz enthalten ist. 

```
   {
      "uri" : "http://purl.org/lobid/lv#containedIn",
      "label" : "Enthalten in",
      "multilangLabel" : {},
      "container" : "@set",
      "name" : "containedIn"
   }
```

`primaryForm`: no documentation
```
  {
      "name" : "primaryForm",
      "container" : "@set",
      "multilangLabel" : {},
      "uri" : "http://purl.org/dc/terms/isFormatOf",
      "label" : "Primärform"
   }
```

`similar`: no documentation

```
   {
      "multilangLabel" : {},
      "label" : "Ähnlich zu",
      "uri" : "http://umbel.org/umbel#isLike",
      "container" : "@set",
      "name" : "similar"
   }
```

`seeAlso`: no documentation
```
   {
      "uri" : "http://www.w3.org/2000/01/rdf-schema#seeAlso",
      "label" : "Weitere Informationen",
      "multilangLabel" : {},
      "name" : "seeAlso",
      "container" : "@set"
   }
```

`hasVersion`: some discussion but no fitting documentation or use. See https://github.com/hbz/lobid-resources/issues/1242#issuecomment-832483535 It needs some better definition and remodelling.

```
   {
      "label" : "Vgl.",
      "uri" : "http://purl.org/dc/terms/hasVersion",
      "multilangLabel" : {},
      "container" : "@set",
      "name" : "hasVersion"
   }
```

`predecessor`: no documentation
```

   {
      "name" : "predecessor",
      "container" : "@set",
      "multilangLabel" : {},
      "uri" : "http://rdaregistry.info/Elements/u/P60261",
      "label" : "Vorgänger"
   }
```

`successor`:  no documentation

```
   {
      "name" : "successor",
      "container" : "@set",
      "multilangLabel" : {},
      "uri" : "http://rdaregistry.info/Elements/u/P60278",
      "label" : "Nachfolger"
   }
```

It is especially important to differentiate between `related` as an very broad concept to `similar`, `primaryForm`, `hasVersion`, `isPartOf`, `containedIn`.

Perhaps I forgot some relations. The documentation would help to improve the transformations of these elements. Since some create duplicate or not fitting info.

unknown relation-elements:

```
   {
      "name" : "source",
      "multilangLabel" : {},
      "label" : "Quelle",
      "uri" : "http://id.loc.gov/ontologies/bibframe/source"
   }
```

```
   {
      "multilangLabel" : {},

      "label" : "übergeordnete Ressource",
      "uri" : "http://purl.org/lobid/lv#hasSuperordinate",
      "name" : "hasSuperordinate",
      "container" : "@set"
   },
```

**Comment by TobiasNx** [comment]
`similar` seems to be controversial: https://github.com/hbz/lobid-resources/search?q=umbel&type=issues

**Comment by acka47** [comment]
There are some high-level semantic web intricacies behind this, see e.g. https://wiki.dnb.de/pages/viewpage.action?pageId=68061169:

> Ist ein Identifier in Form eines HTTP-URIs verfügbar, muss geprüft werden, ob die Property [owl:sameAs](https://wiki.dnb.de/www.w3.org/2002/07/owl#sameAs) verwendet werden kann. Sie ist semantisch sehr ausdrucksstark, denn sie drückt absolute Gleichheit der beiden Ressourcen aus. Das bringt für die Nutzung der RDF-Daten den größten Mehrwert, hat aber verheerende Konsequenzen bei falscher Anwendung.
 Ist nicht zweifelsfrei zu klären, ob ein URI tatsächlich für eine identische Ressource steht oder sie nur zum Thema hat (siehe Beispiele), darf die Aussage der absoluten Gleichheit auf keinen Fall erfolgen. In dieser Situation kann mit den Properties [schema:sameAs](http://schema.org/sameAs) oder [umbel:isLike](http://umbel.org/umbel#isLike) gearbeitet werden. Sie weisen auf die hohe Ähnlichkeit hin, sind jedoch semantisch so weich, dass keine problematischen Konsequenzen bei der Datennutzung (insbesondere beim Reasoning) entstehen . 

I think a pragmatic solution to use https://schema.org/sameAs whereever we have use `owl:sameAs` or `umbel:isLike` before.

**Comment by acka47** [comment]
> did we decided to put everything that was hasVersion and similar before into[ sameAs bases on schema.](https://schema.org/sameAs) ?

I don't think that we made a decision there. Currently, `sameAs` is mapped to `owl:sameAs` in the [JSON-LD context]():

```json
{
   "sameAs":{
      "@id":"http://www.w3.org/2002/07/owl#sameAs",
      "@container":"@set"
   }
}
```
Is `owl:sameAs` already used somewhere in the fix? It'd be ok for me to replace it in the context with `sdo:sameAs` which won't be an API break from the JSON view but from an RDF view. Maybe we should discuss this after the standup today.

**Comment by acka47** [comment]
> Is owl:sameAs already used somewhere in the fix? It'd be ok for me to replace it in the context with sdo:sameAs which won't be an API break from the JSON view but from an RDF view. Maybe we should discuss this after the standup today.

We discussed this and decided to move forward with merging `owl:sameAs` and `umbel:isLike` into `sdo:sameAs` which will be put into the JSON-LD context as URI for `sameAs`.

**Comment by TobiasNx** [comment]
https://github.com/hbz/lobid-resources/issues/1390#issuecomment-1222528834

> @acka47 primaryForm transformation in ALEPH MORPH say it links to primary and secondary form and vice versa. Isn't this contradicting the relation primaryForm and only true for the http://purl.org/dc/terms/isFormatOf which is the URI behind `primaryForm`.
> 
> https://github.com/hbz/lobid-resources/blob/69d0c8b011f83905e0114fc0a79069c86ab1e084/src/main/resources/morph-hbz01-to-lobid.xml#L1469-L1474
> 
> In ALMA I only used the link of the secondary to the primary, not the other way around. okay?

@acka47 
I reopen the question for this. Why do we use primaryForm also as link to the reproduction. It also more narrow than the concept it is based on: http://purl.org/dc/terms/isFormatOf
